### PR TITLE
Check in/72163/video appointment icon labels

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Adds vvc and cvt labels and icons to upcoming appointment list

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
[department-of-veterans-affairs/va.gov-team#72163](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72163)

## Testing done

- Unit
- Visual

## Screenshots

Screenshot 2024-05-13 at 10 06 26 AM

## What areas of the site does it impact?

Check in upcoming appointments list

## Acceptance criteria

- [ ] Content matches wire frames for vvc and ctv appointments.

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by brianseek — COMMENTED** [review]
Code looks great! there is a bit of a display issue on my end that I don't seen in your screenshots. Maybe a browser rendering issue.  The icon text alignment is off
Screenshot 2024-05-13 at 1 53 50 PM
I would suggest using `vads-u-align-items--flex-start` in the container rather than `vads-u-align-items--baseline`
Like so:
Screenshot 2024-05-13 at 1 55 44 PM

**Comment by wullaski** [comment]
@brianseek I fixed that icon spacing I needed to do a yarn install to see the problem. I also adjusted some spacing and added borders with the right color.

File: src/applications/check-in/components/UpcomingAppointmentsList.jsx [hunk 1]
```diff
@@ -86,19 +86,28 @@ const UpcomingAppointmentsList = props => {
...
-                        {appointments.map(appointment => {
+                        {appointments.map((appointment, number) => {
                           return (
-                            <UpcomingAppointmentsListItem
-                              key={getAppointmentId(appointment)}
-                              appointment={appointment}
-                              goToDetails={handleDetailClick}
-                              router={router}
-                            />
+                            <div
```
Comment (brianseek) [inline src/applications/check-in/components/UpcomingAppointmentsList.jsx:96]: this DIV should go inside the <li> the immediate child of a <ul> should be an <li>

File: src/applications/check-in/components/UpcomingAppointmentsListItem.jsx [hunk 2]
```diff
@@ -97,6 +98,7 @@ const UpcomingAppointmentsListItem = props => {
 
 UpcomingAppointmentsListItem.propTypes = {
   appointment: PropTypes.object.isRequired,
+  border: PropTypes.object.isRequired,
```
Comment (brianseek) [inline src/applications/check-in/components/UpcomingAppointmentsListItem.jsx:101]: this should be `border: PropTypes.bool.isRequired,`

**Review by brianseek — APPROVED** [review]
🥇

**Pull request merged**